css/ourHorses.css

```diff
@@ -84,11 +83,12 @@
 	flex-direction: column;
 	width: 260px;
 	align-items: center;
-	background-color: #fff;
-	border: 1px solid #EAEAEA;
+	background-color: var(--color-white);
+	border: 1px solid var(--color-border-white);
 	box-shadow: 0px 14px 30px rgba(0, 0, 0, 0.07);
 	border-radius: 5px;
-	transition: all 1s !important;
+	transition: transform 1s,
```

Comment:
Why do you need important?
try not to use important if there is another opportinities such as specificity

Reply:
Done.

Reply:
Done.